02-install/02-01-install_rust_on_linux.md

```diff
@@ -1,50 +1,52 @@
 #Rust for Linux
 
-Rust 支持主流的操作系统，Linux,Mac和 windows。
+Rust supports mainstream operating systems，Linux,Mac and windows.
 
-Rust 为Linux用户提供了两种安装方式：
+Rust two installation methods for Linux users：
 
-### 1、直接下载安装包：
+### 1、Download the installation package directly：
 
-直接下载安装包的话需要检查一下你当前操作系统是64位还是32位，分别下载对应的安装包。
+To download the installation package directly, you need to check whether your current operating system is 64-bit or 32-bit, and download the corresponding installation package.
 
-查看操作系统请在终端执行如下命令:
+To view the operating system, execute the following command in the terminal:
```

Comment:
specially `uname` command was only works well in linux/macOS .

Reply:
changed